doc/modules/ensemble.rst

```diff
@@ -294,6 +294,12 @@ In practice those estimates are stored as an attribute named
 the value, the more important is the contribution of the matching feature
 to the prediction function.
 
+But, the impurity-based feature importance suffers from being computed 
+on statistics derived from the training dataset. There is a nice alternative
+to the impurity-based feature importance which is :ref:`permutation_importance`.
```

Comment:
:ref:`permutation_importance`. is a nice alternative impurity-based feature importance.